### DISCUSSION TITLE
No import suggestions in vue files

### DISCUSSION THREAD

**Discussion by extrem7** [vuejs/language-tools#1278] [Q&A]
I'm using Volar+TS Vue Plugin.
For some reason, VS Code shows only ts competitive files suggestions in Vue script tag:
![image](https://user-images.githubusercontent.com/24792876/167388822-a55106c6-1132-4157-80ca-84db6a1bd05f.png)

But in ts files everything works nice:
![image](https://user-images.githubusercontent.com/24792876/167388935-312da7a6-3c0e-46f5-9308-00cd0ea15f0e.png)

Where could be the problem?

**Comment by johnsoncodehk** [comment]
Works for me, can you reproduce this problem in v0.40.13 / v1.0.0-alpha?
